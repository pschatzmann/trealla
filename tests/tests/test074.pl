:- initialization(main).

test(G, Error, Context) :- catch(G, error(Error,Context), writeln(ok)).

main :-
	test(term_variables(t,[_,_|a]), E, C),
	write(E), nl,
	write(C), nl.
