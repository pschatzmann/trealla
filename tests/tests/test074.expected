ok
type_error(list,[_17,_18|a])
term_variables/2
